### PR TITLE
{bio}[foss/2015b] Clustal-Omega 1.2.0, argtable 2.13 (OK)

### PR DESCRIPTION
**Pull request by nathanhaigh**
Working on Easyconfigs for Clustal-Omega-1.2.0-foss-2015b

### CODE REVIEW

File: easybuild/easyconfigs/c/Clustal-Omega/Clustal-Omega-1.2.0-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+homepage = 'http://www.clustal.org/omega/'
+description = """ Clustal Omega is a multiple sequence alignment 
+ program for proteins. It produces biologically meaningful multiple 
+ sequence alignments of divergent sequences. Evolutionary relationships 
+ can be seen via viewing Cladograms or Phylograms """
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+dependencies = [('argtable', '2.13')]
```
Comment (boegel) [inline easybuild/easyconfigs/c/Clustal-Omega/Clustal-Omega-1.2.0-foss-2015b.eb]: please move this down, below `sources`/`source_urls`

File: easybuild/easyconfigs/a/argtable/argtable-2.13-foss-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+version = '2.13'
+
+homepage = 'http://argtable.sourceforge.net/'
+description = """ Argtable is an ANSI C library for parsing GNU style 
+ command line options with a minimum of fuss. """
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%s%s.tar.gz' % (name, version.replace('.','-'))]
```
Comment (boegel) [inline easybuild/easyconfigs/a/argtable/argtable-2.13-foss-2015b.eb]: nitpicking: space after `,`

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/a/argtable/argtable-2.13-foss-2015b.eb (hunk 2 above)
Comment (nathanhaigh) [inline easybuild/easyconfigs/a/argtable/argtable-2.13-foss-2015b.eb]: nitpic all you like - I enjoy the fact that someone is taking the time to look so closely at the code!

**Comment by nathanhaigh** [comment]
Test report by @nathanhaigh
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux ubuntu 14.04, Intel(R) Xeon(R) CPU E5-2699 v3 @ 2.30GHz, Python 2.7.6
See https://gist.github.com/4146253fbfebb9c88e3d for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ba95ea8b7114a1f0d476 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @nathanhaigh

**Comment by nathanhaigh** [comment]
@boegel: ready to merge in

**Comment by boegel** [comment]
Thanks for the reminder @nathanhaigh!

**Pull request merged**